show/main.py

```diff
@@ -1818,6 +1819,34 @@ def mmu():
     click.echo(proc.stdout.read())
 
 
+#
+# 'buffer' command ("show buffer")
+#
+@cli.group(cls=clicommon.AliasedGroup)
+def buffer():
+    """Show buffer information"""
+    pass
+
+#
+# 'configuration' command ("show buffer command")
+#
+@buffer.command()
+def configuration():
+    """show buffer configuration"""
+    cmd = "mmuconfig -l"
+    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True)
+    click.echo(proc.stdout.read())
```

Comment:
Why can't we use run_command wrapper?

Reply:
I just wanted to leverage the legacy code and didn't realize it.
Will fix it.

Reply:
Fixed.